src/hooks/useStockExchangeData.js

```diff
@@ -0,0 +1,58 @@
+export default () => (
+  [
+    {
+      name: 'BTC',
+      price: 184406.98,
+      change: 4.9,
+      cap: 3444312.55
```

Comment:
Use builder pattern with uuidv4 as identifier: https://codeburst.io/builder-pattern-in-javascript-e5b13e4e51af